scrape_me.py

```diff
@@ -1,54 +1,67 @@
-proxies = [] # Will contain proxies [ip, port]
+ua = UserAgent()  # From here we generate a random user agent
+proxies = []  # Will contain proxies [ip, port]
 
 # Main function
+
+
 def main():
-  # Retrieve latest proxies
-  proxies_req = Request('https://www.sslproxies.org/')
-  proxies_req.add_header('User-Agent', ua.random)
-  proxies_doc = urlopen(proxies_req).read().decode('utf8')
-
-  soup = BeautifulSoup(proxies_doc, 'html.parser')
-  proxies_table = soup.find(id='proxylisttable')
-
-  # Save proxies in the array
-  for row in proxies_table.tbody.find_all('tr'):
-    proxies.append({
-      'ip':   row.find_all('td')[0].string,
-      'port': row.find_all('td')[1].string
-    })
-
-  # Choose a random proxy
-  proxy_index = random_proxy()
-  proxy = proxies[proxy_index]
-
-  for n in range(1, 100):
-    req = Request('http://icanhazip.com')
-    req.set_proxy(proxy['ip'] + ':' + proxy['port'], 'http')
-
-    # Every 10 requests, generate a new proxy
-    if n % 10 == 0:
-      proxy_index = random_proxy()
-      proxy = proxies[proxy_index]
-
-    # Make the call
-    try:
-      my_ip = urlopen(req).read().decode('utf8')
-      print('#' + str(n) + ': ' + my_ip)
-    except: # If error, delete this proxy and find another one
-      del proxies[proxy_index]
-      print('Proxy ' + proxy['ip'] + ':' + proxy['port'] + ' deleted.')
-      proxy_index = random_proxy()
-      proxy = proxies[proxy_index]
+    # Retrieve latest proxies
+    proxies_req = Request('https://www.sslproxies.org/')
+    proxies_req.add_header('User-Agent', ua.random)
+    proxies_doc = urlopen(proxies_req).read().decode('utf8')
+
+    soup = BeautifulSoup(proxies_doc, 'html.parser')
+    proxies_table = soup.find(
+        "table", {"class": 'table table-striped table-bordered'})
+
+    # Save proxies in the array
+    for row in proxies_table.tbody.find_all('tr'):
+        proxies.append({
+            'ip':   row.find_all('td')[0].string,
+            'port': row.find_all('td')[1].string
+        })
+
+    # Choose a random proxy
+    proxy_index = random_proxy()
+    proxy = proxies[proxy_index]
+
+    for n in range(1, 100):
+        req = Request('http://icanhazip.com')
+        req.set_proxy(proxy['ip'] + ':' + proxy['port'], 'http')
+
+        # Every 10 requests, generate a new proxy
+        if n % 10 == 0:
+            proxy_index = random_proxy()
+            proxy = proxies[proxy_index]
+
+        # Make the call
+        try:
+            my_ip = urlopen(req, timeout=5.0).read().decode('utf8')
+            print('#' + str(n) + ': ' + my_ip)
+        except:  # If error, delete this proxy and find another one
+            del proxies[proxy_index]
+            print('Proxy ' + proxy['ip'] + ':' + proxy['port'] + ' deleted.')
+            proxy_index = random_proxy()
+            proxy = proxies[proxy_index]
+
+    keys = proxies[0].keys()
+    with open('proxy_list.csv', 'w', newline='') as output_file:
```

Comment:
Hey, thank you for the PR!

What's this all about? Why are you saving the proxy list in a file not being used anywhere else?

Reply:
I'm sorry, but I commited without removing the code I added for personal reasons.

Reply:
If you can omit that part I can proceed to test it and merge it 😄  Thank you!